prereise/cli/data_sources/tests/conftest.py

```diff
@@ -0,0 +1,8 @@
+TEXAS_REGION_LIST = ["Texas"]
+STRING_DATE_2021_12_1 = "2021-12-1"
+STRING_DATE_2021_5_1 = "2019-5-1"
```

Comment:
I believe you want to either rename the variable to `STRING_DATE_2019_5_1` or change its value to "2021-5-1"